ui/src/plugins/dev.perfetto.LinuxPerf/index.ts

```diff
@@ -59,19 +59,26 @@ export default class implements PerfettoPlugin {
 
   private async addProcessPerfSamplesTracks(trace: Trace) {
     const pResult = await trace.engine.query(`
-      SELECT DISTINCT upid
+      SELECT DISTINCT upid, pct.name AS cntrName
```

Comment:
Can we guarantee for a given perf_session_id, there will be exactly one timebase `perf_counter_track`?

Reply:
Ack sg.